doc/userdoc/installation/cmake_options.rst

```diff
@@ -28,70 +28,116 @@ Options for configuring NEST
-    -Dcythonize-pynest=[OFF|ON]                  Use Cython to cythonize pynestkernel.pyx.
-                                                 If OFF, PyNEST has to be build from
-                                                 a pre-cythonized pynestkernel.pyx.
-                                                 [default=ON]
-
-Change compilation behavior::
-
-    -Dstatic-libraries=[OFF|ON]     Build static executable and libraries. [default=OFF]
-    -Dwith-optimize=[OFF|ON|<list;of;flags>]       Enable user defined optimizations. Separate
-                                                   multiple flags by ';'.
-                                                   [default OFF, when ON, defaults to '-O3']
-    -Dwith-warning=[OFF|ON|<list;of;flags>]        Enable user defined warnings. Separate
-                                                   multiple flags by ';'.
-                                                   [default ON, when ON, defaults to '-Wall']
-    -Dwith-debug=[OFF|ON|<list;of;flags>]          Enable user defined debug flags. Separate
-                                                   multiple flags by ';'.
-                                                   [default OFF, when ON, defaults to '-g']
-    -Dwith-intel-compiler-flags=[<list;of;flags>]  User defined flags for the Intel compiler.
-                                                   Separate multiple flags by ';'.
-                                                   [defaults to '-fp-model strict']
-    -Dwith-libraries=<list;of;libraries>           Link additional libraries. Give full path.
-                                                   Separate multiple libraries by ';'.
-                                                   [default OFF]
-    -Dwith-includes=<list;of;includes>             Add additional include paths. Give full
-                                                   path without '-I'. Separate multiple include
-                                                   paths by ';'. [default OFF]
-    -Dwith-defines=<list;of;defines>               Additional defines, e.g. '-DXYZ=1'.
-                                                   Separate multiple defines by ';'. [default OFF]
+
+**Change parallelization scheme:**
+
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-mpi=[OFF|ON|</path/to/mpi>]          | Request compilation with MPI. Optionally give directory with   |
+|                                             | MPI installation. [default=OFF]                                |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-openmp=[OFF|ON|<OpenMP-Flag>]        | Enable OpenMP multi-threading. Optional: set OMP flag.         |
++---------------------------------------------+----------------------------------------------------------------+
+
+**Set default libraries:**
```

Comment:
"External libraries"

I would also put the two Python-related entries into a separate table and maybe put that first of all or right after parallelization.

Do python/cython only have ON/OFF or can one specify a python interpreter?